crates/bevy_pbr/src/render/skin.rs

```diff
@@ -188,6 +184,28 @@ impl SkinUniformInfo {
     }
 }
 
+#[derive(Resource, Clone, Debug, ExtractResource, Reflect)]
+#[reflect(Resource, Default, Debug, Clone)]
+pub struct GlobalSkinnedMeshSettings {
+    /// Maximum number of joints supported for skinned meshes.
+    ///
+    /// It is used to allocate buffers.
+    /// The correctness of the value depends on the GPU/platform.
+    /// The default value of `256` is chosen because it is guaranteed to work everywhere (source needed).
+    pub max_joints: usize,
+    /// Max target count available for [morph targets](MorphWeights).
```

Comment:
A similar comment here about the default values and why they were chosen is important here.